src/components/static/Navigation.svelte

```diff
@@ -122,16 +122,16 @@
   </div>
     {#if !isMenuHidden}
       <!-- STANDARD NAVBAR -->
-      <div class="hidden mx-24 my-4 text-white lg:block xl:block 2xl:block h-96">
-        <div class="flex justify-center">
+      <div class="hidden text-white lg:block xl:block 2xl:block h-96" style="margin-left: 12.5%; margin-right: 12.5%;">
```

Comment:
Probably should replace this with `mx-0.5`, its roughly equivalent.

https://tailwindcss.com/docs/margin

Reply:
Updated to keep in line, used slightly different tailwind classes